site/constants/index.ts

```diff
@@ -53,7 +53,14 @@ export const COOKIES_DISRUPTION_INFO = "disruption-info";
 
 export const COOKIES_DISRUPTION_ERRORS = "disruption-errors";
 
+export const COOKIES_ADD_CONSEQUENCE_INFO = "cdd-consequence-info";
+
+export const COOKIES_ADD_CONSEQUENCE_ERRORS = "cdd-consequence-errors";
+
 export const CREATE_DISRUPTION_PAGE_PATH = "/create-disruption";
 
 export const ERROR_PATH = "/_error";
+
+export const ADD_CONEQUENCE_PAGE_PATH = "/add-consequence";
```

Comment:
```suggestion
export const ADD_CONSEQUENCE_PAGE_PATH = "/add-consequence";
```

Reply:
can't this just be hard coded into the page? 
I feel like the path should be /type-of-consequence rather than add as we will be incorporated 3 pages for add consequence